GPU/Common/FramebufferCommon.cpp

```diff
@@ -1752,17 +1752,20 @@ void FramebufferManagerCommon::Resized() {
 	}
 
 	postShaderIsUpscalingFilter_ = shaderInfo ? shaderInfo->isUpscalingFilter : false;
+	postShaderIsSSAAFilter_ = shaderInfo ? shaderInfo->isSSAAFilter : 0;
```

Comment:
Sorry for nagging, but I of course also meant to rename it :)   postShaderSSAAFilterLevel ?

Reply:
Sorry for forgetting about that;p, a holiday-of-too-much-food made me lazy.